netbeans/CheckerFrameworkPlugin/nbproject/private/platform-private.properties

```diff
@@ -0,0 +1 @@
+user.properties.file=C:\\Users\\Tony\\AppData\\Roaming\\NetBeans\\8.1\\build.properties
```

Comment:
The private files indeed shouldn't be in the repository. The rest of the files in `nbproject` should be included, however. Thanks for pointing that out!